dev/create-release/do-release-docker.sh

```diff
@@ -0,0 +1,344 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# Creates a Phoenix release candidate. The script will update versions, tag the branch,
+# build Phoenix binary packages and documentation, and upload maven artifacts to a staging
+# repository. There is also a dry run mode where only local builds are performed, and
+# nothing is uploaded to the ASF repos.
+#
+# Run with "-h" for options. For example, running below will do all
+# steps above using the 'rm' dir under Downloads as workspace:
+#
+# $ ./do-release-docker.sh  -d ~/Downloads/rm
+#
+# The scripts in this directory came originally from spark [1]. They were then
+# modified to suite the hbase context, which were further adopted to Phoenix.
+# These scripts supercedes the old
+# ../make_rc.sh script for making release candidates because what is here is more
+# comprehensive doing more steps of the RM process as well as running in a
+# container so the RM build environment can be a constant.
+#
+# It:
+#  * Tags release
+#  * Sets version to the release version
+#  * Sets version to next SNAPSHOT version.
+#  * Builds, signs, and hashes all artifacts.
+#  * Pushes release tgzs to the dev dir in a apache dist.
+#  * Pushes to repository.apache.org staging.
+#
+# The entry point is here, in the do-release-docker.sh script.
+#
+# 1. https://github.com/apache/spark/tree/master/dev/create-release
+#
+set -e
+
+# Set this to build other phoenix repos: e.g. PROJECT=phoenix-connectors
```

Comment:
Should be in README.txt?

Reply:
I don't think we need to document the environment variables, the important ones (like this one) are already exposed as CLI options.